test/jdk/javax/swing/JTable/KeyBoardNavigation.java

```diff
@@ -224,13 +174,127 @@ public void setValueAt(Object aValue, int row, int column) {
+                  shift and control verifying each change to the selection against
+                  the spec. If it does, press "pass", otherwise press "fail".
+
+                """;
+
+        INSTRUCTIONS += getOSSpecificInstructions();
+        PassFailJFrame.builder()
+                .instructions(INSTRUCTIONS)
+                .rows(11)
+                .columns(50)
+                .testUI(KeyBoardNavigation::initTest)
+                .build()
+                .awaitAndCheck();
+    }
+
+    public static String getOSSpecificInstructions() {
+        final String WINDOWS_SPECIFIC = """
+                Navigate In - Tab, shift-tab
+                Return/shift-return - move focus one cell down/up.
+                Tab/shift-tab -  move focus one cell right/left.
+                Up/down arrow - deselect current selection; move focus one
+                                cell up/down
+                Left/right arrow - deselect current selection; move focus
+                                   one cell left/right
+                PageUp/PageDown - deselect current selection; scroll up/down
+                                  one JViewport view; first visible cell in
+                                  current column gets focus
+                Control-PageUp/PageDown - deselect current selection;
+                                          move focus and view to
+                                          first/last cell in current row
+                Home/end - deselect current selection; move focus and view to
+                           first/last cell in current row
+                Control-home/end - deselect current selection;
+                                   scroll up/down one  JViewport view;
+                                   first/last visible row of the table
+                F2 - Allows editing in a cell containing information without
+                     overwriting the information
+                Esc -  Resets the cell content back to the state it was in
+                       before editing started
+                Ctrl+A, Ctrl+/ - Select All
+                Ctrl+\\ - De-select all
+                Shift-up/down arrow -  extend selection up/down one row
+                Shift-left/right arrow - extend selection left/right one
+                                         column
+                Control-shift up/down arrow -  extend selection to top/bottom
+                                                of column
+                Shift-home/end -  extend selection to left/right end of row
+                Control-shift-home/end  - extend selection to beginning/end
+                                          of data
+                Shift-PageUp/PageDown - extend selection up/down one view
+                                        and scroll table
+                Control-shift-PageUp/PageDown - extend selection left/right
+                                                end of row
+                """;
```

Comment:
I see that for "Navigate In", you mix capitalization for `Tab` and `shift-tab`. Looks a bit off. Maybe better to standardize capitalization for keys. Maybe title-case or all-caps the key names.

Reply:
Sure, I have now Capitalized the Keys.